tests/kafkatest/tests/streams/streams_broker_compatibility_test.py

```diff
@@ -66,14 +67,11 @@ def test_compatible_brokers(self, broker_version):
 
         self.processor.wait()
 
-        num_consumed_mgs = self.consumer.total_consumed()
+        wait_until(lambda: self.consumer.total_consumed() == 1, timeout_sec=30, err_msg="Did expect to read exactly one message but got none within 30 seconds")
```

Comment:
nit:
The error text can be misleading in the case where more than one message was received, the original assert seem to have covered this case (in its error message).
Maybe change the wait to ' total_consumed() > 0' and then keeping the original assert for actual verification and error message?